grouper/fe/templates/group-permission-request.html

```diff
@@ -0,0 +1,100 @@
+{% extends "base.html" %}
+
+{% block heading %}
+    <a href="/groups">Groups</a>
+{% endblock %}
+
+{% block subheading %}
+    Request Permission for Group {{ group.name }}
+{% endblock %}
+
+{% block script %}
+<script type="text/javascript">
+    $(function () {
+        var args_by_perm = {{ args_by_perm_json }};
+
+        var $reason_divs = $('.form-group-reason');
+        var $argument_divs = $('.form-group-argument');
+
+        var $argument_select = $('#dropdown_form #argument');
+        var $argument_text = $('#text_form #argument');
+
+        var $dropdown_form_div = $('#dropdown_form');
+        var $text_form_div = $('#text_form');
+
+        var $perm_fields = $('.input-permission_name');
+        var $reason_fields = $('.input-reason');
+
+        function update_args() {
```

Comment:
Improve the commenting of the various cases in this function. Currently, they just cover the "what"; what's the "why"?